fabfile.py

```diff
@@ -165,9 +169,13 @@ def publish(c: Connection):
 def finish(c):
     pass
 
+# For the following @task functions, "target" is an ignored parameter
+#  It is a workaround to allow for use in using command line arguments
+#  For selecting a "target" in ns.configure
+# Otherwise, fabfile will claim to not recognize it
```

Comment:
When I wrote this fab did not support arguments very well. This seems to have changed recently.
https://docs.fabfile.org/en/1.10/usage/fab.html#per-task-arguments

This may be a better alternative and let us avoid custom argv parsing entirely.

Reply:
Not sure if that's okay with the ns.configure code to call inside each "task" function

The link describes a way to pass arguments into functions, so I probably still have to do arg parsing unless I can call like a "load_target" function and then a "run_deploy" function one after the other (The examples don't show that)

Also, I'm not too experienced with making updates (but please continue to link them so I can take a look in my spare time) for this CompServ system, so hopefully, a future iteration can maybe look into this or when I have time
Need to update, test, and repair SNAFUs if they happen